x/ethbridge/handler.go

```diff
@@ -87,14 +114,17 @@ func handleMsgCreateEthBridgeClaim(
 
 func handleMsgBurn(
 	ctx sdk.Context, cdc *codec.Codec, accountKeeper types.AccountKeeper,
-	bridgeKeeper Keeper, msg MsgBurn,
+	bridgeKeeper Keeper, msg MsgBurn, sugaredLogger *zap.SugaredLogger,
 ) (*sdk.Result, error) {
 	if !bridgeKeeper.ExistsPeggyToken(ctx, msg.Symbol) {
+		sugaredLogger.Errorw("Native token can't be burn.",
+			"token symbol", msg.Symbol)
```

Comment:
Let's have `invalidTokenSymbolForBurn` be the key for this error message instead of "token symbol".